lib/rubocop/cop/style/constant_visibility.rb

```diff
@@ -26,20 +26,43 @@ module Style
       #     public_constant :BAZ
       #   end
       #
+      # @example ExcludeModules: false (default)
+      #   # bad
+      #   class Foo
+      #     MyClass = Struct.new()
+      #   end
+      #
+      #   # good
+      #   class Foo
+      #     MyClass = Struct.new()
+      #     public_constant :MyClass
+      #   end
+      #
+      # @example ExcludeModules: true
+      #   # good
+      #   class Foo
+      #     MyClass = Struct.new()
+      #   end
+      #
       class ConstantVisibility < Base
         MSG = 'Explicitly make `%<constant_name>s` public or private using ' \
               'either `#public_constant` or `#private_constant`.'
 
         def on_casgn(node)
           return unless class_or_module_scope?(node)
           return if visibility_declaration?(node)
+          return if excluded_module?(node)
 
           message = message(node)
           add_offense(node, message: message)
         end
 
         private
 
+        def excluded_module?(node)
```

Comment:
I think that API is misleading, as modules will get excluded when the config is true, but from the function name it seems that we're checking whether some some module is excluded.